resources/assets/components/blocks/ScholarshipInfoBlock/ScholarshipInfoBlock.js

```diff
@@ -56,12 +60,22 @@ const ScholarshipInfoBlock = ({
   numberOfScholarships,
   utmLabel,
 }) => {
-  const { loading, error, data } = useQuery(SCHOLARSHIP_AFFILIATE_QUERY, {
-    variables: {
-      utmLabel,
-      preview: env('CONTENTFUL_USE_PREVIEW_API'),
-      campaignId,
-    },
+  let loading;
+  let error;
+  let data;
+
+  if (utmLabel) {
+    ({ loading, error, data } = useQuery(SCHOLARSHIP_AFFILIATE_QUERY, {
+      variables: {
+        utmLabel,
+        preview: env('CONTENTFUL_USE_PREVIEW_API'),
+        campaignId,
+      },
+    }));
+  }
+
+  const { loading2, error2, data2 } = useQuery(SCHOLARSHIP_INFO_QUERY, {
```

Comment:
This is [destructuring an object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) that the `useQuery` hook responds – so it's actually looking for properties named `loading2`, `error2`, and `data2` on the object (and likely not finding them!). Here's what [that response object](https://www.apollographql.com/docs/react/api/react-hooks/#result) looks like.

You probably want to rename these instead, like this:

```suggestion
  const { loading: scholarshipLoading, error: scholarshipError, data: scholarshipData } = useQuery(SCHOLARSHIP_INFO_QUERY, {
```

Reply:
@DFurnes You are a genius! That works, thanks! And that makes total sense, it couldn't find the value of the prop because I renamed the props instead of just giving the props an alternate value.